extensions/wikia/Email/Controller/CategoryAddController.class.php

```diff
@@ -58,6 +65,41 @@ protected function assertValidPageAddedToCategory() {
 		}
 	}
 
+	public function assertCanEmail() {
+		parent::assertCanEmail();
+		$this->assertEmailsNotThrottled();
+	}
+
+	private function assertEmailsNotThrottled() {
+		global $wgMemc;
+		$emailsSent = $wgMemc->get( $this->getTargetUserCacheKey() );
+		if ( $emailsSent != false && $emailsSent['sent'] >=  self::EMAILS_PER_THROTTLE_PERIOD ) {
```

Comment:
Please use strict type comparison.
Additionally I would consider checking if `$emailsSent` is valid and contains right keys.